Wikipedia/Code/SearchResultsViewController.swift

```diff
@@ -123,5 +124,12 @@ class SearchResultsViewController: ArticleCollectionViewController {
         collectionView.backgroundColor = theme.colors.midBackground
     }
 
+    @objc func updateVisibleCells() {
```

Comment:
@mcleinman Works great! Just a small suggestion - I noticed other listeners of this notification inspect the article from the notification and exit early if it's not something the screen is interested in. Currently all cells on this screen will get reloaded for any article that changes throughout the app, whether it's on this screen or not. Can you change it to something like this so we're only configuring the cell of the updated article?

```
@objc func updateArticleCell(_ notification: NSNotification) {
        guard
            let updatedArticle = notification.object as? WMFArticle,
            let updatedArticleKey = updatedArticle.inMemoryKey
        else {
            return
        }
        
        guard updatedArticle.hasChangedValuesForCurrentEventThatAffectSavedState else {
            return
        }
        
        for indexPath in collectionView.indexPathsForVisibleItems {
            guard articleURL(at: indexPath)?.wmf_inMemoryKey == updatedArticleKey else {
                continue
            }
            
            guard let cell = collectionView.cellForItem(at: indexPath) as? ArticleRightAlignedImageCollectionViewCell else {
                continue
            }
            
            configure(cell: cell, forItemAt: indexPath, layoutOnly: false)
        }
    }
```

Reply:
Good catch. Just added this (and combined some of the `guard`s. Let me know if you think there are any more tweaks I should make!